[2025-03-14T19:00:01.910+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T19:00:02.006+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T18:55:00+00:00 [queued]>
[2025-03-14T19:00:02.010+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T18:55:00+00:00 [queued]>
[2025-03-14T19:00:02.010+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T19:00:02.016+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform> on 2025-03-14 18:55:00+00:00
[2025-03-14T19:00:02.020+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=398) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T19:00:02.021+0000] {standard_task_runner.py:72} INFO - Started process 399 to run task
[2025-03-14T19:00:02.022+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'transform', 'scheduled__2025-03-14T18:55:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/open_weather_dag.py', '--cfg-path', '/tmp/tmpls4ollmh']
[2025-03-14T19:00:02.022+0000] {standard_task_runner.py:105} INFO - Job 117: Subtask transform
[2025-03-14T19:00:02.043+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.transform scheduled__2025-03-14T18:55:00+00:00 [running]> on host a8e3300004fc
[2025-03-14T19:00:02.078+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lherrera5034@gmail.com' AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T18:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T18:55:00+00:00'
[2025-03-14T19:00:02.078+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T19:00:02.092+0000] {logging_mixin.py:190} INFO - Data trasnformed
[2025-03-14T19:00:02.097+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T19:00:02.101+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T19:00:02.101+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=transform, run_id=scheduled__2025-03-14T18:55:00+00:00, execution_date=20250314T185500, start_date=20250314T190002, end_date=20250314T190002
[2025-03-14T19:00:02.124+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T19:00:02.134+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T19:00:02.135+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
